src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -374,47 +376,21 @@ public Boolean visitDeclared(AnnotatedDeclaredType type, Void p) {
         }
     };
 
-    /**
-     * TODO: Similar but not the same as AnnotatedTypeFactory.constructorFromUse with space set aside from
```

Comment:
Some comment about why this is overridden would still be good.

Reply:
Okay, updated!